package.json

```diff
@@ -12,18 +12,18 @@
         "@alpinejs/focus": "^3.14.9",
         "@alpinejs/persist": "^3.14.9",
         "@tailwindcss/forms": "^0.5.10",
+        "@tailwindcss/postcss": "^4.1.8",
         "@tailwindcss/typography": "^0.5.16",
         "@types/alpinejs": "^3.13.11",
         "alpinejs": "^3.14.9",
         "laravel-vite-plugin": "^1.2.0",
-        "tailwindcss": "^3.4.17"
+        "tailwindcss": "^4.1.8"
     },
     "devDependencies": {
-        "@typescript-eslint/eslint-plugin": "^8.28.0",
-        "@typescript-eslint/parser": "^8.28.0",
         "@blumilksoftware/eslint-config": "^3.0.0",
         "@types/alpinejs__focus": "^3.13.4",
-        "autoprefixer": "^10.4.21",
+        "@typescript-eslint/eslint-plugin": "^8.28.0",
+        "@typescript-eslint/parser": "^8.28.0",
         "postcss": "^8.5.3",
         "vite": "^6.0.11"
```

Comment:
Could you update Vite, to close Renovate PR?

Reply:
Done :blush: